docs/entities/Dev_Install.txt

```diff
@@ -0,0 +1,249 @@
+  - open PowerShell as Administrator
+  - enter `dism.exe /online /enable-feature /featurename:Microsoft-Windows-Subsystem-Linux /all /norestart`
+  - enter `dism.exe /online /enable-feature /featurename:VirtualMachinePlatform /all /norestart`
+  - reboot
+  - open PowerShell as Administrator
+  - enter `wsl --set-default-version 2`
+
+3. install Ubuntu latest from Store app
+  - https://www.microsoft.com/en-us/p/ubuntu-20044-lts/9mttcl66cpxj
+  - pin to Taskbar
+
+4. start Ubuntu shell (from Start or Cortana search)
+  - first time takes a few minutes to unpack everything
+  - enter a username and password that you'll remember
+
+5. download package information
+  - enter `sudo apt update`
+
+6. install latest package versions
+  - enter `sudo apt upgrade`
+
+7. install compiler for Python, etc for BE work
+  - enter `sudo apt install build-essential`
+  - enter `sudo apt install libbz2-dev libffi-dev libreadline-dev libsqlite3-dev libssl-dev zlib1g-dev`
+
+8. install Python + pip for BE work
+  - enter `curl https://pyenv.run | bash`
+  - follow instructions to update .bashrc
+  - enter `source .bashrc`
+  - enter `pyenv install 3.8.5`
+  - enter `pyenv global 3.8.5`
+  - enter `sudo apt install python3-pip`
+  - enter `pip install --upgrade pip`
+  - enter `sudo apt install python3.8-venv`
+
+9. install Node Version Manager + Node.js (latest version) (optional?)
+  - enter `curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.1/install.sh | bash`
+  - enter `source .bashrc`
+  - enter `nvm install node`
+
+10. configure git
+  - enter `git config --global user.name "First Last"`
+  - enter `git config --global user.email "First.Last@gov.bc.ca"`
+  - enter `git config --global credential.helper "/mnt/c/Program\ Files/Git/mingw64/libexec/git-core/git-credential-manager.exe"`
+  - enter `git config --global alias.co checkout`
+  - enter `git config --global alias.br branch`
+  - enter `git config --global alias.ci commit`
+  - enter `git config --global alias.st status`
+  - enter `git config --global core.editor vim`
+  - install ssh files to ~/.ssh (instructions depend on tools used)
+
+11. install Openshift Console (oc) app
+  - see https://docs.openshift.com/container-platform/3.11/cli_reference/get_started_cli.html
+  - https://github.com/openshift/origin/releases
+  - fetch the gz file below and copy it to your Ubuntu home folder
+    - NB: Ubuntu folder is at \\wsl$
+  - enter `mkdir ~/bin`
+  - enter `tar -zxvf openshift-origin-client-tools-v3.11.0-0cbc58b-linux-64bit.tar.gz`
+  - enter `cp openshift-origin-client-tools-v3.11.0-0cbc58b-linux-64bit/oc ~/bin`
+  - enter `rm -rf openshift-origin-client-tools-v3.11.0-0cbc58b-linux-64bit*`
+
+
+*** ALSO NEED TO LOGIN TO NPM FOR PUBLISHING PACKAGES: `npm login`
+
+MORE THINGS FOR UI DEVELOPMENT
+------------------------------
+install global packages and local dependencies
+```
+sudo apt install npm
+sudo npm i -g @vue/cli  // don't need jshint tslint typescript

```

Comment:
I also think we can remove this as we don't use Vue CLI anymore. We're using Vite now.

Reply:
Fixed, thanks!